setup.py

```diff
@@ -32,6 +32,8 @@
 
 packages = [
     'oauth2client',
+    'oauth2client.contrib',
```

Comment:
Interesting, I had no idea you had to explicitly declare sub-modules... something here seems weird.